### ISSUE TITLE
"Forgot Password" request doesn't work on mobile Device

### ISSUE THREAD

**Issue by WillemBielefeld-GEA** [FriendsOfSymfony/FOSUserBundle#2940]
**Symfony 2.x with FOSUserBundle v2.1.2 **:
### **Description of the problem**
We are using a Login Screen build with your Fos UserBundle on a WebApp used by the Browser and also used as a mobile device App.
The mobile App uses an emulated chrome Browser and runs the WebApp in a similar way.

Within our login form there is a "forgot password" Url , which runs the Controller for the FOS resetting Request:

```
<a href="{{ path('fos_user_resetting_request') }}" class="forgot-link">
 {{ 'security.login.forgot_password'|trans({},'general') }} //The translations are custom in an own 'general.language.yml' file and work totaly fine
</a>
```

For whatever Reason it works with no issues in the Browser, but on mobile i experience an error:
"An error occured please try again later."

The routing runs the standard Controller out of the vendor bundle of yours and calls this method:
```
    public function requestAction()
    {
        return $this->render('@FOSUser/Resetting/request.html.twig');
    }
```
The twig file works fine with Chrome, Firefox and IE, there are no errors/bugs i was able to find.
I wasnt able to find out, why this doesnt work within a mobile App.

**Comment by stof** [comment]
Well, with only a message `An error occured please try again later.`, there is nothing I can do to help you. You  would need to check your log to see what the error is.

We don't have any browser-specific things in FOSUserBundle itself, and I'm using this in a webview without issue.

**Comment by WillemBielefeld-GEA** [comment]
I just found the issue myself. I apologize for taking your time.

Maybe the solution might help others aswell:
within our project we used an additional bundle for handling different devices with different templates.
These files are located parallel to the view folder and are named "themes"
Within this themes folder, there was an additional version for "phone" which overrides the original twig file. There were bugs causing this error.

I hope this information will be usefull for others aswell.
